cuebot/scheduler-sim/inject_parity.py

```
@@ -0,0 +1,143 @@
+import os, sys, time, subprocess
+import grpc
+_HERE = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, os.path.join(_HERE, "opencue_proto"))
+sys.path.insert(0, _HERE)
+import job_pb2, job_pb2_grpc
+import farm_spec as spec
+
+DURATION = int(sys.argv[1]) if len(sys.argv) > 1 else 180
+MODE = sys.argv[2] if len(sys.argv) > 2 else "unknown"
+PSQL = spec.psql_cmd()
+OUT_DIR = "/tmp/scheduler-sim"
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/scheduler-sim"

(S108)